content/projects/gsoc/students.adoc

```diff
@@ -252,10 +252,10 @@ Use the community bonding to:
 
 == Coding periods
 
-Students are expected to...
+GSoC contributors are expected to...
 
 . Work on the GSoC project as it is a full-time employment.
- * It means that 30..40 hours per week is an **expected** workload though it can be adjusted upon the agreements with mentors.
+ * It means 12 weeks of writing good code, about 30..40 hours per week is an **expected** workload though it can be adjusted upon the agreements with mentors.
```

Comment:
I thought that they only expected half time work, not full time work during the twelve weeks.
```suggestion
 * It means 12 weeks of writing good code, about 20..30 hours per week is an **expected** workload though it can be adjusted upon the agreements with mentors.
```